test/test_nn.py

```diff
@@ -230,7 +232,8 @@ def fw_hook(inc, h_module, input, output):
             self.assertIsInstance(output, Variable)
             self.assertTrue(h_module is module)
             self.assertEqual(input[0].data, torch.ones(5, 5))
-            self.assertEqual(output.data, torch.Tensor(5, 5).fill_(1 / (1 + 1 / math.e)))
+            self.assertEqual(
```

Comment:
Can you please leave the lines as they were? We accept up to 120 chars in line and they're not very long.